mullvad.js

```diff
@@ -97,35 +94,37 @@ var MullvadVPN = GObject.registerClass({
     // Use Soup.Session to GET am.i.mullvad.net/json
     // The result is used to decide whether or not we're connected to Mullvad
     _fetchConnectionInfo(callback) {
-        const request = new Soup.Message({
-            method: 'GET',
-            uri: Soup.URI.new('https://am.i.mullvad.net/json'),
-        });
+        const message = Soup.Message.new('GET', 'https://am.i.mullvad.net/json');
```

Comment:
Is there a reason to change the variable name? I don't mind if it's more standard to use `message`, but I've never seen it before.

Reply:
I took the name from the GJS docs. They use `message` in this example: https://gjs.guide/extensions/upgrading/gnome-shell-43.html#soup3